packages/admin/cms-admin/src/dashboard/widgets/LatestBuildsDashboardWidget.tsx

```diff
@@ -15,40 +16,39 @@ type Row = {
 };
 
 export type LatestBuildsDashboardWidgetProps = {
-    tableQuery: Pick<TableQueryProps, "api" | "loading" | "error"> & {
-        tableData?: Partial<ITableProps<Row>> & Pick<ITableProps<Row>, "data" | "totalCount">;
-    };
+    data: Partial<DataGridProps<Row>>;
```

Comment:
Is it a good idea to allow passing all DataGrid props to the widget?

Reply:
Yeah, that's probably not necessary 👍🏼